packages/core/src/app/checkout/CheckoutApp.tsx

```diff
@@ -4,10 +4,12 @@ import React, { Component } from 'react';
 import ReactModal from 'react-modal';
 
 import { AnalyticsProvider } from '@bigcommerce/checkout/analytics';
-import '../../scss/App.scss';
+import { ExtensionProvider } from '@bigcommerce/checkout/checkout-extension';
 import { getLanguageService, LocaleProvider } from '@bigcommerce/checkout/locale';
 import { CheckoutProvider } from '@bigcommerce/checkout/payment-integration-api';
 
+import '../../scss/App.scss';
```

Comment:
Is this intentional?

Reply:
Yes, I was suprised linter didn't pick it up before.